Ruby-Hot-Chili-Peppers/MightyPawn
# Scope stuff

Hi guys!

To test my code you should do the following:
Create games in your local database....some with only a white_player, and others with both a white and black player
i.e. game.create(id: 1, white_player_id: 1, black_player_id: 2)
i.e. game.create(id: 2, white_player_id: 1)

You should be able to call Game.available and get a list of the games that have the black player spot = nil.

The Game id's of these available game should display on the static index page when you open the webview.

Furthermore, I have added the ability for the current user to join a game. This is working! When you click on the game id on the homepage it will take you to the show page for that game. However, I forgot how to refresh the  page once you click the Join button... so you have to manually refresh the page for the button to go away and check the rails console to make sure that the player was added.

I'm tired and will find the refresh thing tomorrow (or one of you can remind me what it is :) )
